contracts/NestedAsset.sol

```diff
@@ -14,15 +14,24 @@ contract NestedAsset is ERC721Enumerable, OwnableFactoryHandler {
 
     Counters.Counter private _tokenIds;
 
-    /// @dev Stores the URI of each asset
-    mapping(uint256 => string) private _tokenURIs;
+    /// @dev Base URI (API)
+    string public baseUri;
+
+    /// @dev Token URI when not revealed (not using API)
+    string public unrevealedTokenUri;
+
+    /// @dev NFT contract URI
+    string public contractUri;
 
     /// @dev Stores the original asset of each asset
     mapping(uint256 => uint256) public originalAsset;
 
     /// @dev Stores owners of burnt assets
     mapping(uint256 => address) public lastOwnerBeforeBurn;
 
+    /// @dev True if using the API
```

Comment:
💅 "using the API" is not very explicit. Consider rewording this
Same for L20

Reply:
👍